tests/manual-tests/lib/jquery/plugins/selectbox/VERSION.md

```diff
@@ -0,0 +1,5 @@
+The version of the jquery.selectbox plugin included in this folder was created from the master branch of the fluid-project fork:
+
+https://github.com/fluid-project/jquery.selectbox
```

Comment:
Can you add the link at the README of this repository to include the url of the original github repository?